aws/policy/security-services.yaml

```diff
@@ -110,3 +110,20 @@ Statement:
     Resource:
       - arn:aws:iam::{{ aws_account_id }}:role/ansible_lambda_role
       - arn:aws:iam::{{ aws_account_id }}:role/ansible-test-sts-*
+  - Sid: AllowACMRestrictable
+    Effect: Allow
+    Action:
+      - acm:ImportCertificate
+      - acm:DescribeCertificate
+      - acm:GetCertificate
+      - acm:AddTagsToCertificate
+      - acm:DeleteCertificate
+    Resource:
+      - arn:aws:acm:us-east-1:{{ aws_account_id }}:certificate/*
+  - Sid: AllowACMRestrictable
```

Comment:
Looks like these two could go in the `AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees` Sid instead.

Reply:
Do you want me to merge `AllowACMRestrictable` into `AllowGlobalResourceRestrictedActionsWhichIncurNoFees`? Why? Isn't it better to have more fine grained permissions, rather than allow the `acm:*` actions on `arn:aws:iam::{{ aws_account_id }}:role/*`, and the other way around?